cli/default-reporter/test/__snapshots__/reportingUpdateCheck.ts.snap

```diff
@@ -27,3 +27,17 @@ exports[`print update notification if the latest version is greater than the cur
    ╰──────────────────────────────────────────────────────────────────╯
 "
 `;
+
+exports[`print update notification that suggests to use the standalone scripts for the upgrade 1`] = `
+"
+   ╭──────────────────────────────────────────────────────────────────────────╮
+   │                                                                          │
+   │                    Update available! 10.0.0 → 11.0.0.                    │
+   │       Changelog: https://github.com/pnpm/pnpm/releases/tag/v11.0.0       │
+   │   Run one of the standalone scripts from: https://pnpm.io/installation   │
+   │                                                                          │
+   │          Follow @pnpmjs for updates: https://twitter.com/pnpmjs          │
+   │                                                                          │
+   ╰──────────────────────────────────────────────────────────────────────────╯
+"
+`;
```

Comment:
Need four spaces to match snapshots

Reply:
Locally the test is passing. So I am not sure what is the problem.